src/main/java/org/jfree/svg/util/RyuFloat.java

```diff
@@ -0,0 +1,345 @@
+/*
+ * Copyright 2018 Ulf Adams
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jfree.svg.util;
+
+import java.math.BigInteger;
+
+import static java.lang.Math.min;
+
+/**
+ * An implementation of Ryu for float.
+ */
+public final class RyuFloat {
```

Comment:
Delete this class if there are no usages of float to string conversions in the code base.

Reply:
Fixed.